function main():
	try:
		raise $exception()
	except:
		raise
	return 'ok'
+++exception USER 3
